agent/agent.go

```diff
@@ -53,6 +53,12 @@ func NewWithConfig(cfg Config) (*Agent, error) {
 		return nil, errors.Wrap(err, "could not create a Checker instance")
 	}
 
+	res, err := checker()
+	if err != nil {
+		log.Fatal("Failed to do checks: ", err)
+	}
+	res.PrettyPrint(os.Stdout)
+
```

Comment:
Hmm, I think this block is misplaced. We should not execute the checker when we instantiate a new Agent! What I suggested earlier is to put this in the `runOnce` command handler, copying the `Checker` instantiation from here.